pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sphinx_gherkindoc"
-version = "3.2.7"
+version = "3.2.8"
```

Comment:
Suggestion that we dink a more substantial part of the version number, so make it 3.3.0 instead.
This isn't a backwards breaking compatibility, but it isn't just a small change either (the lines of code, not so many but the capability is pretty substantial)